paper.tex

```diff
@@ -80,7 +80,7 @@ \section{Introduction}
 This is our general term for the central RSE team throughout this paper.
 These RSE Hubs can take the form of, \eg{} full RSE units, smaller RSE groups, Open Source Program Offices (OSPOs), virtually across multiple units or combined under single leadership,
  depending on the environment of the hosting research organisation.
-All of these implementations are considered, taking into account the large variety of research environments in Germany.
+All of these implementations are considered, taking into account the large variety of research environments in Germany which include not only universities, but also research consortia and other research performing organisations.
```

Comment:
Why "consortia" and not "centers"? What are examples for "research consortia"?

Reply:
I was thinking about how to include sth. like Helmholtz, which now has centralied RSE services for their decentralized institutes. Or the other end, something like Transregios, or clusters of excellence that span multiple locations. Because in my little world, there is now such thing as a decentralized center...

Reply:
Maybe "research associations"? Then I guess/hope it's clear what is meant here.